### DISCUSSION TITLE
architecture/redis/01-Redis%E7%BC%93%E5%AD%98%E7%A9%BF%E9%80%8F%E5%8E%9F%E7%90%86%E3%80%81%E5%AE%9E%E6%88%98%E5%8F%8A%E4%BC%98%E5%8C%96

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#663] [Announcements]
# architecture/redis/01-Redis%E7%BC%93%E5%AD%98%E7%A9%BF%E9%80%8F%E5%8E%9F%E7%90%86%E3%80%81%E5%AE%9E%E6%88%98%E5%8F%8A%E4%BC%98%E5%8C%96

缓存穿透是什么 查询信息的时候，如果缓存中存在就返回，不存在就得去查数据库。如果数据库中不存在（例如id为999的用户不存在），一直访问999这个用户就会一直去越过redis查询数据库，这个就是缓存穿透。

http://localhost:8080/architecture/redis/01-Redis%E7%BC%93%E5%AD%98%E7%A9%BF%E9%80%8F%E5%8E%9F%E7%90%86%E3%80%81%E5%AE%9E%E6%88%98%E5%8F%8A%E4%BC%98%E5%8C%96.html

**Comment by hsxleo** [comment]
q
